### DISCUSSION TITLE
Start button becomes unresponsive

### DISCUSSION THREAD

**Discussion by BrianJ20** [valinet/ExplorerPatcher#144] [General]
![bug](https://user-images.githubusercontent.com/92690930/137644012-69c5596a-5c81-418d-ace8-ff85667b209b.png)

When i launch the "File Explorer" from the Start Menu, the Start Button stops responding to a left click.  Right click still works.
This can only be resolved by restarting file explorer process

The File Explorer button is the one added in Personalization -> Start -> Folders.

Is this happening to anyone else or is it just me ?

**Comment by valinet** [comment]
It's happening to me as well, idk why, I will investigate it and hopefully come up with a fix.

**Reply by valinet** [reply]
Issue here: https://github.com/valinet/ExplorerPatcher/issues/145
